docs/one-click-integrations/jira-integration.md

```diff
@@ -0,0 +1,42 @@
+# Jira integration
+
+Pulse integrates with Jira to receive data about issues, necessary to calculate the metrics [Lead time and Cycle time](../metrics/lead-cycle-time.md).
```

Comment:
We can briefly mention that our integration works with Jira **Cloud** (and, since we're at it, do the same for GitHub Cloud). What do you think?

```suggestion
Pulse integrates with Jira Cloud to receive data about issues, necessary to calculate the metrics [Lead time and Cycle time](../metrics/lead-cycle-time.md).
```